todo_app/templates/index.html

```diff
@@ -7,21 +7,51 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
+  <div class="row" style="display: flex;text-align: center;">
+    <div style="width: 45%;">
+      <h2>Not Started</h2>
         {% for task in tasks %}
-          <li>{{ task.title }}</li>
+          {% if task.status == "Not Started" %}
```

Comment:
Really minor but here you have a couple of examples of "magic strings" that are easy to typo (e.g. incorrect capitalisation) or forget to change later - would switching this for a property/method on the Item class called "is_not_started" be neater?

Reply:
Of course! No idea why I didn't do that in the first place